src/Application/Actions/Donations/Create.php

```diff
@@ -147,8 +150,20 @@ protected function action(Request $request, Response $response, array $args): Re
                 $donation->setCampaign($campaign);
             }
 
+            if ($forNewPaymentElement) {
+                // We don't need to set specific payment methods here, they are set on the client side by the payment
+                // element.
+                $stripeMethodProperties = [
+                    'automatic_payment_methods' => [
+                    'enabled' => true,
+                    'allow_redirects' => 'always',
+                    ]
+                ];
+            } else {
```

Comment:
but note that 153 and 163 do continue to be covered by the existing tests. If 153 was the wrong way round 6 tests would error.